Userland/Libraries/LibGUI/Window.h

```diff
@@ -150,9 +150,9 @@ class Window : public GUI::Object {
     void set_main_widget(Widget*);
 
     template<class T, class... Args>
-    inline ErrorOr<NonnullRefPtr<T>> set_main_widget(Args&&... args)
+    inline NonnullRefPtr<T> set_main_widget(Args&&... args)
     {
-        auto widget = TRY(T::try_create(forward<Args>(args)...));
+        auto widget = T::construct(forward<Args>(args)...);
```

Comment:
This means that all GML widgets can't be constructed directly through this function anymore, since they only define try_create. However, since most GML widgets define some custom setup logic anyways, I don't think it's a problem unless you do so. (CI should catch that if it actually *is* a problem.)

Reply:
I don't think this is a big problem because calling `set_main_widget<Foo>()`  for a class generated by the GML compiler will generate a compiler error, which also happens for classes not generated by the GML compiler.